src/adhanpy/calculation/CalculationParameters.py

```diff
@@ -51,70 +51,70 @@ def __init__(
-                self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
-
-            case CalculationMethod.KUWAIT:
-                self.fajr_angle = 18.0
-                self.isha_angle = 17.5
-
-            case CalculationMethod.QATAR:
-                self.fajr_angle = 18.0
-                self.isha_interval = 90
-
-            case CalculationMethod.SINGAPORE:
-                self.fajr_angle = 20.0
-                self.isha_angle = 18.0
-                self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
-
-            case CalculationMethod.UOIF:
-                self.fajr_angle = 12.0
-                self.isha_angle = 12.0
-
-            case CalculationMethod.OTHER:
-                self.fajr_angle = 0.0
-                self.isha_angle = 0.0
+        if calculation_method == CalculationMethod.MUSLIM_WORLD_LEAGUE:
+            self.fajr_angle = 18.0
+            self.isha_angle = 17.0
+            self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
+
+        elif calculation_method == CalculationMethod.EGYPTIAN:
+            self.fajr_angle = 19.5
+            self.isha_angle = 17.5
+            self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
+
+        elif calculation_method == CalculationMethod.KARACHI:
+            self.fajr_angle = 18.0
+            self.isha_angle = 18.0
+            self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
+
+        elif calculation_method == CalculationMethod.UMM_AL_QURA:
+            self.fajr_angle = 18.5
+            self.isha_interval = 90
+
+        elif calculation_method == CalculationMethod.DUBAI:
+            self.fajr_angle = 18.2
+            self.isha_angle = 18.2
+            self.method_adjustments = PrayerAdjustments(0, -3, 3, 3, 3, 0)
+
+        elif calculation_method == CalculationMethod.MOON_SIGHTING_COMMITTEE:
+            self.fajr_angle = 18.0
+            self.isha_angle = 18.0
+            self.method_adjustments = PrayerAdjustments(0, 0, 5, 0, 3, 0)
+
+        elif calculation_method == CalculationMethod.NORTH_AMERICA:
+            self.fajr_angle = 15.0
+            self.isha_angle = 15.0
+            self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
+
+        elif calculation_method == CalculationMethod.KUWAIT:
+            self.fajr_angle = 18.0
+            self.isha_angle = 17.5
+
+        elif calculation_method == CalculationMethod.QATAR:
+            self.fajr_angle = 18.0
+            self.isha_interval = 90
+
+        elif calculation_method == CalculationMethod.SINGAPORE:
+            self.fajr_angle = 20.0
+            self.isha_angle = 18.0
+            self.method_adjustments = PrayerAdjustments(0, 0, 1, 0, 0, 0)
+
+        elif calculation_method == CalculationMethod.UOIF:
+            self.fajr_angle = 12.0
+            self.isha_angle = 12.0
+
+        elif calculation_method == CalculationMethod.OTHER:
+            self.fajr_angle = 0.0
+            self.isha_angle = 0.0
```

Comment:
Can you just keep the changes you made in the night_portions method and in the README. I have something else for the calculation method that I'd like to push once I merge your changes. Thanks.

Reply:
What advantage is there to removing these changes from the PR? This is a functional change allowing the Python version to decrease and without it the version support change is wrong. Why not just merge these changes and make your refactor on top of the new master?